examples/example-app.yaml

```diff
@@ -52,3 +52,17 @@ spec:
         - "/main"
         - "--process-metrics"
         - "--go-metrics"
+---
+apiVersion: monitoring.googleapis.com/v1
+kind: PodMonitoring
+metadata:
+  name: prom-example
+  labels:
+    app.kubernetes.io/name: prom-example
+spec:
+  selector:
+    matchLabels:
+      app: prom-example
```

Comment:
Should this selector be `app` or `app.kubernetes.io/name`? It does not match the actual label name. Note, the `Deployment` uses `app`.

Reply:
I went with `app.kubernetes.io/name` for everything, as that's convention in our other [examples](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.6.0-rc.0/examples/aerospike/pod-monitoring.yaml#L20).